examples/dynamic/views/Html.jsx

```diff
@@ -0,0 +1,33 @@
+var React = require('react');
+var Content = React.createFactory(require('./Content'));
+
+module.exports = React.createClass({
+
+  render: function() {
+    var data = this.props.data;
+
+    // render the content as a dynamic react component
+    var contentHtml = React.renderToString(Content(data));
```

Comment:
I would personally avoid the `createFactory` call and then just make this `renderToString(<Content {...data} />)`, like you have in `main`